lib/linux_admin/network_interface.rb

```diff
@@ -148,7 +148,7 @@ def start
     #
     # @return [Boolean] whether the command succeeded or not
     def stop
-      Common.run(Common.cmd("ifdown"), :params => [@interface]).success?
+      Common.run(Common.cmd("ifconfig"), :params => [@interface, "down"]).success?
```

Comment:
I found `ip a` and `ifconfig` are living in two world. If you type `ifconfig` after a `ifconfig down`, `eth0` has disappeared. But in `ip a`, it's still here, same as you.
![screenshot_manageiq-20170714_2017-07-17_15 56 50](https://user-images.githubusercontent.com/13259400/28287535-0c36157c-6b0a-11e7-8c8a-3af95b2712c8.png)